docs/CHANGELOG.rst

```diff
@@ -21,6 +21,8 @@ Changed
 - Use ``resolwebio/rnaseq:4.5.0`` Docker image in processes that call
   Salmon tool (``library-strandedness``, ``feature_counts`` and
   ``qorts-qc``)
+- Implement dropdown menu for ``upload-bedpe`` process
+- Add validation stringency parameter to ``bqsr`` process
```

Comment:
State that this input parameter is now propagated from the WES pipeline as well.